roles/deprecated_api/README.md

```diff
@@ -0,0 +1,73 @@
+# redhatci.ocp.deprecated_api – Detect To-Be-Removed Kubernetes (K8s) APIs
+
+- [Synopsis](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/deprecated_api/README.md#synopsis)
+- [Requirements](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/deprecated_api/README.md#requirements)
+- [Parameters](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/deprecated_api/README.md#parameters)
+- [Examples](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/deprecated_api/README.md#examples)
+- [Return Values](https://galaxy.ansible.com/ui/repo/published/redhatci/ocp/content/role/deprecated_api/README.md#return-values)
+
+# Synopsis
+
+- Connect to a running K8s cluster with an application workload already deployed.
+- Utilize the community.kubernetes.k8s_info module to access K8s APIs.
+- Parse workload APIs using the APIRequestCount object.
+- Detect to-be-removed APIs by filtering through APIRequestCount.status.removedInRelease.
+- Output a map from OpenShift versions to a list of workload APIs to be removed in each version.
+- Generate a JUnit file containing the testsuite 'Workload Compatibility with OCP Versions.'
+
+
+# Requirements
+
+The role requires the following:
+
+- Install `community.kubernetes.k8s_info` to access K8s APIs.
+- Ensure the availability of the K8s [APIRequestCount](https://github.com/openshift/cluster-kube-apiserver-operator/blob/master/bindata/assets/kube-apiserver/apiserver.openshift.io_apirequestcount.yaml) object. For OpenShift, APIRequestCount is included in the standard distribution from OCP 4.9 onwards. For other distributions, deploy the required CRD and CR before using the role.
+- Python version 3.6 or higher is needed to parse the APIs.
+
+
+# Parameters
+
+| Parameter               | Choices/Defaults      | Comments                                             |
+|-------------------------|-----------------------|------------------------------------------------------|
+| `kubeconfig`            | `~/.kube/config.json` | Path to an existing Kubernetes config file to be used by k8s_info dependency. If not provided, the openshift client will attempt to load the default configuration file from `~/.kube/config.json`. Can also be specified via `K8S_AUTH_KUBECONFIG` environment variable.                                                                                                |
+| `deprecated_namespaces` | `undefined`           | List of namespaces the role will check for deprecated APIs. Typically, it should include the namespaces where you deploy the workload. If left undefined, the role will check all namespaces, excluding those starting with the `openshift` and `kube-` prefixes.                                                |
+| `ocp_version`           | `"4.9"/"4.10"/etc`    | OpenShift version currently running on your cluster. |
+| `job_logs`              | `undefined`           | The third parameter has a default value.    |
```

Comment:
I don't really understand the comment for this variable. I'd extend here a little bit more.

Reply:
sorry, WIP PR, let me push the latest version